### ISSUE TITLE
setValue() throws unexpected error

### ISSUE THREAD

**Issue by vojtys** [vuejs/test-utils#1878]
I'm migrating from vue 2 to version 3 and all tests where I use setValue on inputs return an error:

```Error: wrapper.setValue() cannot be called on input```

The last word changes depending on what type of input setValue I use.

Simple example:

ContactUsForm.vue

```
...
<textarea v-model="form.message" id="contact-message"></textarea>
...

```

ContactUsForm.spec.js
```
 it('validates form before submit', async () => {
        wrapper = mount(ContactUsForm);

        await wrapper.find('form #contact-message').setValue('Short msg');
   
        wrapper.find('form').trigger('submit.prevent');

        expect(wrapper.findAll('.validation-error-message').length).toBe(1);
});
```

Console error: 
```
Error: wrapper.setValue() cannot be called on textarea
```

In vue-test-utils source code i see: 

```
 DOMWrapper.prototype.setValue = function (value) {
        var element = this.element;
        var tagName = element.tagName;
        var type = this.attributes().type;
 ....
        }
        else if (tagName === 'INPUT' || tagName === 'TEXTAREA') {
            element.value = value;
            this.trigger('input');
            // trigger `change` for `v-model.lazy`
            return this.trigger('change');
        }
        else {
            throw Error("wrapper.setValue() cannot be called on ".concat(tagName));
        }
    };
```

tagName is lowercase i my case. Why? 

**Related information:**
- `@vue/test-utils` version: 2.2.3 
- `Vue` version: 3.1.5
- `@vue/compat` version: 3.1.5
- `node` version: v14.18.2
- `npm` (or `yarn`) version: 6.14.15

**Comment by cexbrayat** [comment]
Hi @vojtys 

Can you provide us a small repro online using https://stackblitz.com/edit/vitest-dev-vitest-qoiwy3?file=package.json&initialPath=__vitest__ ?

**Comment by vojtys** [comment]
Hi @cexbrayat, I'm trying to reproduce the test with my project's dependencies, but I can't run the test. I have no experience with the stackBlits app

[https://stackblitz.com/edit/node-aiu835?file=resources/js/app.js](https://stackblitz.com/edit/node-aiu835?file=resources/js/app.js)

**Comment by cexbrayat** [comment]
@vojtys If we can't run the tests, we won't be able to help you, sorry.

Instead of creating a new Stackblitz, you can use the link I shared https://stackblitz.com/edit/vitest-dev-vitest-qoiwy3?file=package.json&initialPath=__vitest 
It's already set up to run tests, you just have to replace the components and tests with yours. 

And please try to build a minimal reproduction: it's harder to understand what happens when there are so many dependencies 😉

**Comment by cexbrayat** [comment]
@vojtys Did you manage to reproduce your issue with a minimal setup on stackblitz?

**Comment by vojtys** [comment]
@cexbrayat yes, with minimal setup on stackblitz works correctly. After a few days of agonizing, I found out that I broke the tests myself. Due to a workaround for another issue.

`SVGElement is not defined` issue I fixed with this line: `global.SVGElement = global.Element;`. This workaround broke my tests.  `global.SVGElement = window.SVGElement;` could be the correct workaround.

I hope I didn't break something else with it, but it works for now. Maybe it will help someone, because I see that the issue `SVGElement is not defined`  is repeated around vue tests.

We can close this issue. Thank you very much.
